fastlane/Fastfile

```diff
@@ -55,21 +55,23 @@ platform :ios do
     crashlytics(
       crashlytics_path: './Pods/Crashlytics/',
       groups: options[:group],
-      notifications: 'true'
+      notifications: 'true',
+      notes: options[:notes]
```

Comment:
You need to handle the case where the user doesn't want notes. Perhaps a Rubyist would be able to come up with an elegant way to handle passing the parameter or not.